impeller/renderer/render_target.h

```diff
@@ -19,28 +19,39 @@ class Context;
 
 class RenderTarget {
  public:
+  struct AttachmentConfig {
+    StorageMode storage_mode;
+    LoadAction load_action;
+    StoreAction store_action;
+  };
+
+  struct AttachmentConfigMSAA {
+    StorageMode storage_mode;
+    StorageMode resolve_storage_mode;
+    LoadAction load_action;
+    StoreAction store_action;
+  };
+
   static RenderTarget CreateOffscreen(
       const Context& context,
       ISize size,
       const std::string& label = "Offscreen",
-      StorageMode color_storage_mode = StorageMode::kDevicePrivate,
-      LoadAction color_load_action = LoadAction::kClear,
-      StoreAction color_store_action = StoreAction::kStore,
-      StorageMode stencil_storage_mode = StorageMode::kDeviceTransient,
-      LoadAction stencil_load_action = LoadAction::kClear,
-      StoreAction stencil_store_action = StoreAction::kDontCare);
+      AttachmentConfig color_attachment_config =
```

Comment:
Maybe hoist this up to `kDefaultAttachmentConfig` (ditto for MSAA)?

Reply:
Done.